TransformProcessor/qSlicerTransformProcessorModuleWidget.cxx

```diff
@@ -256,6 +260,7 @@ void qSlicerTransformProcessorModuleWidget::setSignalsBlocked( bool newBlock )
   d->inputInitialTransformComboBox->blockSignals( newBlock );
```

Comment:
Use qsignalblocker for each widget, we should never overwrite block state without recording the original value (and later restoring the original value).

Reply:
See new issue #171